packages/gatsby-cli/src/reporter/loggers/ink/components/messages.js

```diff
@@ -1,4 +1,4 @@
-import React from "../react"
+import React from "react"
```

Comment:
intentionally importing "react" here (it's wrong), to check eslint rule to avoid using "react" import directly